### PR TITLE
feat(@desktop/wallet): Wallet settings - Notification after each action

### LINKED ISSUE #11719: Wallet settings - Notification after each action

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #11719

### What does the PR do

Fixes some errors I found + notifications for updating network url + notification for include watch only address

### Affected areas

Wallet Settings

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://github.com/status-im/status-desktop/assets/60327365/b544a8c6-6237-48d2-9137-db22f59652b0

### CODE REVIEW

File: src/app/modules/main/profile_section/wallet/networks/module.nim [hunk 1]
```diff
@@ -98,6 +98,7 @@ method toggleTestNetworksEnabled*(self: Module) =
 
 method updateNetworkEndPointValues*(self: Module, chainId: int, newMainRpcInput, newFailoverRpcUrl: string) =
   self.controller.updateNetworkEndPointValues(chainId, newMainRpcInput, newFailoverRpcUrl)
+  self.refreshNetworks()
```
Comment (saledjenic) [inline src/app/modules/main/profile_section/wallet/networks/module.nim:101]: @Khushboo-dev-cpp you're refreshing networks here regardless to the result of the action in previous step. Would be much better if you would connect to signal `SIGNAL_NETWORK_ENDPOINT_UPDATED` emitted from the previous action and refresh network only in that case.

File: src/app_service/service/network/service.nim [hunk 2]
```diff
@@ -89,6 +96,13 @@ proc getNetwork*(self: Service, chainId: int): NetworkDto =
     if chainId == net.chainId:
         return net
 
+proc getNetworkByChainId*(self: Service, chainId: int): NetworkDto =
```
Comment (saledjenic) [inline src/app_service/service/network/service.nim:99]: @Khushboo-dev-cpp this function doesn't include test/prod as param?
I mean for the same chainId you will always get prod network, never test network.

File: src/app_service/service/settings/service.nim [hunk 3]
```diff
@@ -47,6 +47,9 @@ type
   SettingProfilePictureArgs* = ref object of Args
     value*: int
 
+  SettingsIncludeWatchOnlyAccountsArgs* = ref object of Args
```
Comment (saledjenic) [inline src/app_service/service/settings/service.nim:50]: @Khushboo-dev-cpp please name this type more general (like one we have in the line 36), e.g. `SettingsBoolValueArgs` that way we can send the same type whenever we need to send `bool` param and the name of signal is used to differ where that signal comes from.

**Review by saledjenic — CHANGES_REQUESTED** [review]

File: src/app_service/service/network/service.nim (hunk 2 above)
Comment (saledjenic) [reply src/app_service/service/network/service.nim:99]: @Khushboo-dev-cpp cool, thanks, I didn't know that, I've thought that for example arbitrum and arbitrum test network have the same chain id.

Comment (Khushboo-dev-cpp) [reply src/app_service/service/network/service.nim:99]: Aah I can understand, but yes they are different chainIds fir arb and arb test. Ive addressed the other comments from you btw

**Review by alaibe — APPROVED** [review]

**Pull request merged**